assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -148,6 +152,26 @@ function useCanvasKeys(ref) {
     [arrangeSelection]
   );
 
+  // Edit mode
+  useGlobalKeyDownEffect(
+    'enter',
+    () => {
+      if (selectedElements.length === 0) {
```

Comment:
Should this be `selectedElements.length !== 1`? Otherwise we'd be entering editing mode randomly in a multiselection.